test/test-pr-544.ts

```diff
@@ -0,0 +1,93 @@
+
+        return mm.parseFile(filePath).then(metadata => {
+            testEqualsSame(metadata);
+
+            t.strictEqual(metadata.common.originalalbum, 'Original Album', 'metadata.common.originalalbum');
+
+            // t.strictEqual(metadata.common.showMovement, true, 'metadata.common.showMovement'); // Not exists in MP3 (Written as comment)
+            // t.strictEqual(metadata.common.work, 'Work', 'metadata.common.work'); // Not exists in MP3 (Written as comment)
+
+            // t.strictEqual(metadata.common.hdVideo, 2, 'metadata.common.hdVideo'); // Not exists in MP3 (Written as comment)
+            // t.strictEqual(metadata.common.stik, 9, 'metadata.common.stik'); // Not exists in MP3 (Written as comment)
+        });
+    });*/
+
+    it("mp3-id3v24", () => {
+        const filename = 'mp3/pr-544-id3v24.mp3';
+        const filePath = path.join(samplePath, filename);
+
+        return mm.parseFile(filePath).then(metadata => {
+            testEqualsSame(metadata);
+
+            t.strictEqual(metadata.common.originalalbum, 'Original Album', 'metadata.common.originalalbum');
+
+            // t.strictEqual(metadata.common.showMovement, true, 'metadata.common.showMovement'); // Not exists in MP3 (Written as comment)
+            // t.strictEqual(metadata.common.work, 'Work', 'metadata.common.work'); // Not exists in MP3 (Written as comment)
+
+            // t.strictEqual(metadata.common.hdVideo, 2, 'metadata.common.hdVideo'); // Not exists in MP3 (Written as comment)
+            // t.strictEqual(metadata.common.stik, 9, 'metadata.common.stik'); // Not exists in MP3 (Written as comment)
+        });
+    });
+
+    it("mp4", () => {
+        const filename = 'mp4/pr-544.m4a';
+        const filePath = path.join(samplePath, filename);
+
+        return mm.parseFile(filePath).then(metadata => {
+            testEqualsSame(metadata);
+
+            // t.strictEqual(metadata.common.originalalbum, 'Original Album', 'metadata.common.originalalbum'); // Not exists in MP4 (Written as comment)
+
+            t.strictEqual(metadata.common.showMovement, true, 'metadata.common.showMovement');
+            t.strictEqual(metadata.common.work, 'Work', 'metadata.common.work');
+
+            t.strictEqual(metadata.common.hdVideo, 2, 'metadata.common.hdVideo');
+            t.strictEqual(metadata.common.stik, 9, 'metadata.common.stik');
+        });
+    });
+
+    function testEqualsSame(metadata: mm.IAudioMetadata): void {
+        t.strictEqual(metadata.common.albumartistsort, 'Album Artist Sort', 'metadata.common.albumartistsort');
+        t.strictEqual(metadata.common.albumsort, 'Album Sort', 'metadata.common.albumsort');
+        t.strictEqual(metadata.common.artistsort, 'Artist Sort', 'metadata.common.artistsort');
+        t.deepEqual(metadata.common.composersort, ['Composer Sort'], 'metadata.common.composersort'); // TODO: Why composer sort field is an array but other sort fields not?
```

Comment:
@Borewit 
Why composer sort field is an array but other sort fields not?

Reply:
Hmm, I have no idea. Good point.

Reply:
@Borewit 
Should I change to make `composersort` not `multiple` (=additional breaking change) or keep it as is?

Both works in my case

Reply:
Please correct it, and add it to the list of changes: _metadata.common.composersort has been changed to a singleton_